googletest/include/gtest/gtest-typed-test.h

```diff
@@ -283,12 +292,20 @@ INSTANTIATE_TYPED_TEST_SUITE_P(My, FooTest, MyTypes);
     static bool gtest_##TestName##_defined_ GTEST_ATTRIBUTE_UNUSED_ = \
         GTEST_TYPED_TEST_SUITE_P_STATE_(SuiteName).AddTestName(       \
             __FILE__, __LINE__, GTEST_STRINGIFY_(SuiteName),          \
-            GTEST_STRINGIFY_(TestName));                              \
+            GTEST_STRINGIFY_(TestName),                               \
```

Comment:
Do we want to manipulate the tag string that we store internally? Maybe store tags in all caps? If I understand correctly, someone could tag a test with (for instance) "Thread", and someone else could tag another test with "thread", resulting in two different tags. Unless we're really diligent with tag formatting, we could end up accidentally skipping tests. But, then again, maybe that's not a huge concern.

Reply:
I left it the same as the test name. If you mess that up you don't run the test.

Reply:
Good deal. Makes sense to me.